index.html

```diff
@@ -355,7 +368,7 @@ <h3 class="supporting-text">Let's connect</h3>
         <ul class="skill-list"></ul>
       </section>
     </section>
-    <footer>
+    <footer id="contact">
```

Comment:
- [ ] [OPTIONAL] I would like to recommend you use a tag like `<div>` instead of `<footer>`, the <footer> tag is used to define a footer of an HTML document. This section contains the footer information (author information, copyright information, carriers, etc). :+1:

Reply:
**Great, okay I will kindly do that , I though since it's the ''footer" of the page it would be semantic @codecaiine thanks again**😄

Reply:
@codecaiine 
I will change it back actually beacause according to the requirement of the previous milestone you have to put it in a `footer`, kindly find the link here, and read under project requirementsm thank you.
https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md